pyanaconda/ui/tui/spokes/__init__.py

```diff
@@ -166,7 +166,11 @@ def prompt(self, entry = None):
             return _("Enter new value for '%s' and press enter\n") % entry.title
 
     def input(self, entry, key):
-        if entry.aux.match(key):
+        not_allowed_dirs = ['etc', 'boot', 'opt', 'proc', 'lib64', 'sbin', 'mnt',
+                            'lib', 'var', 'lost+found', 'sys', 'usr',
+                            '.esmtp_queue', 'dev', 'tmp', 'home', 'srv', 'root',
+                             'run', 'bin', 'media', 'daemon']
```

Comment:
This list should definitely live somewhere else so that it can be used from other places in code as well. For example, how are we checking this in GUI?